### PR TITLE
Add several useful packages to components.json

### PR DESCRIPTION
**Pull request by lachlancollins**
This is just a collection of packages I've used or starred on github that I noticed were missing :)

Packages:

- @tanstack/svelte-table
- @histoire/plugin-svelte
- lucia
- @monaco-auth/sveltekit
- trpc-svelte-query
- svelte-legos
- @supabase/auth-helpers-sveltekit
- svelte-markdown
- svelte-exmarkdown
- @sentry/svelte
- svelte-french-toast
- @zerodevx/svelte-toast

### CODE REVIEW

File: src/routes/components/components.json [hunk 1]
```diff
@@ -2469,5 +2469,101 @@
...
+		"tags": ["components and libraries", "layout and structure", "typescript"]
+	},
+	{
+		"title": "Histoire",
+		"npm": "@histoire/plugin-svelte",
+		"repository": "https://github.com/histoire-dev/histoire",
+		"description": "Fast and beautiful interactive component playgrounds, powered by Vite",
+		"category": "Testing",
+		"tags": ["testing"]
+	},
```
Comment (MacFJA) [inline src/routes/components/components.json:2483]: It is a component or a tool ?

File: src/routes/components/components.json [hunk 2]
```diff
@@ -2469,5 +2469,101 @@
...
+		"category": "Display Components",
+		"tags": ["components and libraries", "markdown"]
+	},
+	{
+		"title": "Sentry for Svelte",
+		"npm": "@sentry/svelte",
+		"repository": "https://github.com/getsentry/sentry-javascript",
+		"description": "Official Sentry SDKs for JavaScript",
+		"category": "Integration",
+		"tags": ["integrations", "error monitoring"]
```
Comment (MacFJA) [inline src/routes/components/components.json]: `error monitoring` is a new tag, and this package is the only using it

File: src/routes/components/components.json (hunk 1 above)
Comment (lachlancollins) [reply src/routes/components/components.json:2483]: I would say both, however Storybook is also in `components.json` which is why I put it in the same location. It also does have the `Hst` component you import into your story files.

File: src/routes/components/components.json (hunk 2 above)
Comment (lachlancollins) [reply src/routes/components/components.json]: I've removed this tag!

**Comment by lachlancollins** [comment]
@MacFJA I've fixed the merge conflicts here :)

**Pull request merged**